.github/workflows/tests.yml

```
@@ -3,12 +3,11 @@ on:
   push:
     branches:
       - main
-  pull_request:
-    types: [opened, reopened]
-    paths:
-      - 'packages/**'
-      - '!docs/**'
-      - '!website/**'
+  pull_request: {}
```

Do you maybe want to reverse the behavior to be in sync with the monorepo?

```suggestion
  pull_request:
        paths-ignore:
          - 'docs/**'
          - 'website/**'
```

---

there's actually a problem in the monorepo where if you add something to docs in the same PR as other code, all the tests skip... So i think i'd rather be over-interested in the test suite running – with that context, wdyt?

---

Yeah we'd have to introduce the same skip actions here too then I suppose. Whatever you prefer: just wanted to throw the idea out :)